core/src/main/java/cz/xtf/core/config/XTFConfig.java

```diff
@@ -29,11 +29,21 @@
 
 	private static final Properties properties = new Properties();
 
+	private static final String XTF_TEST_PROPERTIES_CONFIG_PROPERTY_NAME = "config.xtf.test.properties.path";
```

Comment:
`XTF_TEST_PROPERTIES_CONFIG_PROPERTY_NAME` -> `TEST_PROPERTIES_PATH`. Same for global test properties. `XTF` and `CONFIG` are redundant in here. `PROPERTY_NAME` by itself doesn't say anything. Also it will be consistent will all other config classes.

`config.xtf.test.properties.path` -> `xtf.test_properties.path`. Similar reasons. `config` is redundant in here and not used anywhere else. Same way all xtf properties start with `xtf`. `test_properties` rather then `test.properties` so it's clear on first sight that this is property related to this particular file. Same for global test properties.

Reply:
I was worried about that `xtf.test.properties.path` would conflict with the `xtf.test.*` namespace, `xtf.test_properties.path` didn't occur to me